frame/assets/src/lib.rs

```diff
@@ -468,6 +468,10 @@ pub mod pallet {
 		///
 		/// Emits `Destroyed` event when successful.
 		///
+		/// NOTE: It can be helpful to first freeze an asset before destroying it so that you
+		/// can provide accurate witness information and prevent users from manipulating state
+		/// in a way that can make it harder to destroy.
+		///
```

Comment:
Should we also note that for assets with many holders and approvals in storage the owner may need to call `burn` and `force_cancel_approval` so the weight of `destroy` can fit into a block?